src/Context/HelperContext.php

```diff
@@ -148,29 +160,38 @@ public function takeScreenshotAfterFailedStep(AfterStepScope $scope) {
       $featureFolder = str_replace(' ', '', $scope->getFeature()->getTitle());
 
       // Get the name of the failed test.
-      $scenarioName = $this->currentScenario->getTitle();
-      $failedTest = str_replace(' ', '', $scenarioName);
+      $failedTest = str_replace(' ', '', $this->scenarioName);
```

Comment:
Shouldn't we apply the same sanitization here too so we don't have failed file saves due to incompatible names?